ftw/simplelayout/browser/client/dist/simplelayout.js

```diff
@@ -635,6 +635,13 @@ define('app/simplelayout/Block',[
       return this;
     };
 
+    this.markLinks = function() {
```

Comment:
Ok. I wouldn't have put it into the core block functionality, but I'm ok with keeping it there.
@maethu is it ok for you?